tasks/install_from_repo.yml

```diff
@@ -13,4 +14,25 @@
     path: /etc/redis
     state: directory
     mode: 0755
-    owner: "{{ redis_user }}"
\ No newline at end of file
+    owner: "{{ redis_user }}"
+
+# Remove systemd service from package installation
+# systemd service will be created in server.yml
+- name: stop redis service from package
+  block:
+    - name: Stop redis-server service
+      service:
+        name: "redis-server"
+        state: stopped
+
+    - name: Remove systemd service file
+      file:
+        path: "/lib/systemd/system/redis-server.service"
+        state: absent
+
+    - name: Reload systemd daemon
+      command: systemctl daemon-reload
+
+  when:
+    - ansible_os_family == "Debian"
+    - _redis_repo_install.changed
```

Comment:
I think, this will cause issues when a user later tries to update or uninstall the redis application.

IMO, it is better to have a variable `redis_service_name`, which is set to `redis-server` or `redis_6379` depending on `redis_install_from_source` value.

And, most of the tasks in `server.yml` should be skipped (except "create redis config file").
When config file is updated, the service must be restarted or reloaded.

Reply:
I agree, I had initially created the config file, log directory and pid directory in this task  and skipped `tasks/server.yml` all together, 

 https://github.com/miarec/ansible-redis/blob/b06fa96e1188e14ad5ae5d231de3c19ff423189a/tasks/install-package.yml

However I changed it to match the the source repository.

I will update this.